website/docs/cloud-docs/workspaces/index.mdx

```diff
@@ -48,7 +48,7 @@ Both Terraform Cloud and Terraform CLI have features called workspaces, but they
 
 ## Listing and Filtering Workspaces
 
-Click **Workspaces** in the top navigation bar. Terraform Cloud shows a list of all workspaces in the current organization.
+Click **Workspaces**. Terraform Cloud shows a list of all workspaces in the current organization.
 
 ![Screenshot: the list of workspaces](/img/docs/index-list.png)
```

Comment:
this is another screenshot that's not in the page that's currently live....
https://developer.hashicorp.com/terraform/cloud-docs/workspaces